multitenancy-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/multitenancy/autoconfig/AxonServerTenantProvider.java

```diff
@@ -174,15 +174,15 @@ private TenantDescriptor toTenantDescriptor(ContextOverview context) {
         return new TenantDescriptor(context.getName(), metaDataMap);
     }
 
-    protected void addTenant(TenantDescriptor tenantDescriptor) {
+    public void addTenant(TenantDescriptor tenantDescriptor) {
```

Comment:
When making the API public, please provide some javadoc to explain the purpose and side-effects of this function.